packages/webapp/components/layouts/AccountLayout/index.tsx

```diff
@@ -44,10 +66,18 @@ export default function AccountLayout({
       </Head>
       <NextSeo {...Seo} noindex nofollow />
       <main className="relative mx-auto flex w-full flex-1 flex-row items-stretch pt-0 laptop:max-w-[calc(100vw-17.5rem)]">
-        <SidebarNav
-          className="absolute z-3 ml-auto h-full w-full border-l border-theme-divider-tertiary bg-background-default tablet:relative tablet:w-[unset]"
-          basePath="account"
-        />
+        {isMobile ? (
+          <ProfileSettingsMenu
+            isOpen={isOpen}
+            logout={logout}
+            onClose={closeSideNav}
+          />
+        ) : (
+          <SidebarNav
```

Comment:
Ah was hoping we could remove it or uniform it.
Maybe check with design for a follow up task to uniform it? (non urgent)

Reply:
Yeah, at first I actually deleted this completely thinking we won't need it anymore, but then on laptop/tablet, they have a completely different design/components.

> Maybe check with design for a follow up task to uniform it? (non urgent)

Sure, will send a message.